### PR TITLE
Add package.json

### PR DESCRIPTION
**Pull request by AndreGCRamos**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+  "name": "ecosia-privacy-policy",
+  "version": "1.0.0",
+  "description": "",
```
Comment (m90) [inline package.json]: I think a small description would be nice for the public.

File: package.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  "main": "de.md",
```
Comment (m90) [inline package.json]: What happens when you `require` the package now? Maybe it'd be better to omit this?

File: package.json [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/ecosia/ecosia-privacy-policy.git"
+  },
+  "author": "Ecosia Devs",
+  "license": "ISC",
```
Comment (m90) [inline package.json]: We should check w/ product what license to pick here as it might have some serious and unwanted implications.

Comment (AndreGCRamos) [reply package.json]: Yeah, sounds like a good idea.

Comment (AndreGCRamos) [reply package.json]: Do you have any recommendation for now?

Comment (m90) [reply package.json]: The easiest thing to do would probably making this `UNLICENSED`, which means feel free to look at it, but it is ours and we do not grant any rights.

File: package.json [hunk 4]
```diff
@@ -0,0 +1,18 @@
+{
+  "name": "ecosia-privacy-policy",
+  "version": "1.0.0",
+  "description": "Ecosia's Privacy Policy in multiple languages using markdown",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```
Comment (m90) [inline package.json]: Since this is no code, I would actually remove this. Otherwise it looks like a big glaring TODO.

**Review by m90 — DISMISSED** [review]
:woman_judge:

**Review by m90 — APPROVED** [review]
:man_judge:

**Comment by m90** [comment]
@AndreGCRamos as this is public facing, can we also add a short README here so visitors feel welcome and know what it's about?

**Comment by AndreGCRamos** [comment]
Ok, will add that

**Pull request merged**